src/app/tests/suites/DL_LockUnlock.yaml

```diff
@@ -49,6 +49,38 @@ tests:
       response:
           value: 1
 
+    - label: "Verify that lock state attribute value is set to Locked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 1
+
+    - label: "Try to unbolt the door without PIN"
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 10000
+
+    - label: "Verify that lock state attribute value is set to Unlocked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 2
+
+    - label: "Try to lock the door without a PIN"
+      command: "LockDoor"
+      timedInteractionTimeoutMs: 10000
+
+    - label: "Verify that lock state attribute value is set to Locked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 1
+
+    - label: "Verify that lock state attribute value is set to Locked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 1
```

Comment:
This is duplicating the immediately preceding test, right?

Reply:
Yes, you're right.
I've duplicated the "Lock" + "Check lock state" after the Unbolt to have it in lock state again for the upcoming tests but somehow duplicated the "readAttribute" during that.